tests/integration_tests/evaluation/comparison/test_embedding.py

```diff
@@ -0,0 +1,98 @@
+from typing import Tuple
+
+import numpy as np
+import pytest
+
+from langchain.evaluation.comparison.embedding import (
+    EmbeddingDistance,
+    PairwiseEmbeddingStringEvalChain,
+)
+
+
+@pytest.fixture
+def vectors() -> Tuple[np.ndarray, np.ndarray]:
+    """Create two random vectors."""
+    np.random.seed(0)
+    vector_a = np.random.rand(10)
```

Comment:
Non-deterministic tests can be hard to manage.

Reply:
Yupp going to remove this randomness